sources/packages/api/src-sql/DisbursementReceipts/Create-disbursement-receipts-table.sql

```diff
@@ -0,0 +1,64 @@
+--Create table sims.disbursement_receipts.
+CREATE TABLE IF NOT EXISTS sims.disbursement_receipts(
+    id SERIAL PRIMARY KEY,
+    batch_run_date DATE NOT NULL,
+    student_sin CHAR(9) NOT NULL,
+    disbursement_schedule_id INT NOT NULL REFERENCES sims.disbursement_schedules(id),
+    funding_type CHAR(2) NOT NULL,
+    total_entitled_disbursed_amount NUMERIC(5) NOT NULL,
+    total_disbursed_amount NUMERIC(5) NOT NULL,
+    disburse_date DATE NOT NULL,
+    disburse_amount_student NUMERIC(5) NOT NULL,
+    disburse_amount_institution NUMERIC(5) NOT NULL,
+    date_signed_institution DATE NOT NULL,
+    institution_code CHAR(4) NOT NULL,
+    disburse_method_student CHAR(1) NOT NULL,
+    study_period_end_date DATE NOT NULL,
+    -- Audit columns
+    created_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+    updated_at TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+    creator INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
+    SET
+        NULL,
+        modifier INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
+    SET
+        NULL,
+        CONSTRAINT disbursement_schedule_id_funding_type_unique UNIQUE (disbursement_schedule_id, funding_type)
+);
+
+-- ## Comments
+COMMENT ON TABLE sims.disbursement_receipts IS 'Disbursement receipts for the disbursements sent.';
+
+COMMENT ON COLUMN sims.disbursement_receipts.id IS 'Auto-generated sequential primary key column.';
+
+COMMENT ON COLUMN sims.disbursement_receipts.batch_run_date IS 'Effective date of data on the disbursement receipt file';
```

Comment:
From where does the name `batch_run_date` come? Is it anywhere in the documentation?